modules/prometheus-exporter_varnish/conf/readme.yaml

```
@@ -0,0 +1,25 @@
+documentations:
+  - name: Prometheus Exporter for Varnish
+    url: https://github.com/jonnenauha/prometheus_varnish_exporter
+
+source_doc:
```

There's an error because of the missing pipe : 

```suggestion
source_doc: |
```